python/ray/tune/utils/util.py

```diff
@@ -152,6 +153,17 @@ def flush(self, *args, **kwargs):
         self.stream2.flush(*args, **kwargs)
 
 
+def env_integer(key, default):
```

Comment:
is there any reason not move this into ray_constants in this PR?

Reply:
Good question - I wanted to get this PR in without needing to wait for Ray CI + and didnt want to risk breaking ray core.